DESCRIPTION

```diff
@@ -11,7 +11,7 @@ Description: Provides easier interaction with
     Users can upload data to 'Socrata' portals directly
     from R.
 Version: 1.7.7-1
```

Comment:
This will have to be 1.7.8 (we've already released 1.7.7).

Reply:
Yeah, I forgot to pull the master branch down to my local computer, which is why I had three commits for what should have been one commit. 

I can update the version number.